felix/bpf-gpl/bpf.h

```diff
@@ -261,7 +261,7 @@ CALI_CONFIGURABLE_DEFINE(ext_to_svc_mark, 0x4b52414d) /*be 0x4b52414d = ASCII(MA
 CALI_CONFIGURABLE_DEFINE(psnat_start, 0x53545250) /* be 0x53545250 = ACSII(PRTS) */
 CALI_CONFIGURABLE_DEFINE(psnat_len, 0x4c545250) /* be 0x4c545250 = ACSII(PRTL) */
 CALI_CONFIGURABLE_DEFINE(flags, 0x00000001)
-
+CALI_CONFIGURABLE_DEFINE(host_tunnel_ip, 0x4c4e5554) /* be 0x4c4e5554 = ACSII(TUNL) */
```

Comment:
For my information, `0x4c4e5554 = ACSII(TUNL)` and similar ones are to set the default value?

Reply:
Ah, got it. It is used by `PatchHostTunnelIPv4()`

Reply:
yup! Just a magic number that se search for, hoping it does not clash with anything else :sweat_smile: It is a relict for ut tests, the libbpf configuration is much better and safer.